.editorconfig

```diff
@@ -7,6 +7,10 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.java]
```

Comment:
The `yarn build` task does some automatic refactoring of the code, and one of the changes it does is tabs-to-spaces. So might as well have code be written with spaces by default.
(Personally, my preference is tabs, but I don't think I care enough to pick a fight about it.)

Reply:
This is possibly caused by our addition of googleformat to gradle build. I'm okay with either. Though I suppose switching back to tabs is going to create a large CL.